src/main/java/org/repodriller/filter/diff/DiffFilter.java

```diff
@@ -0,0 +1,17 @@
+package org.repodriller.filter.diff;
+
+/**
+ * Filter out unwanted Diffs based on file extensions.
```

Comment:
This class operates on file names, not `Diff`s.

Reply:
You're right, but conceptually speaking, the purpose is to filter out Diffs. That's why I say "...based on file extensions".

Reply:
See my comment above.